operatorcourier/nest.py

```diff
@@ -52,7 +52,7 @@ def nest_bundles(yaml_files, registry_dir, temp_registry_dir):
         if not os.path.exists(csv_folder):
             os.makedirs(csv_folder)
 
-        csv_path = '%s/%s.clusterserviceversion.yaml' % (csv_folder, csv_name)
```

Comment:
I have strong preferrence to use `os.path.join()` for manipulation with filenames.
```python
csv_path = os.path.join(csv_folder, '{}.clusterserviceversion.yaml'.format(csv_name) )
```

Reply:
... or `Path` from `pathlib` because this is pure py3
https://docs.python.org/3/library/pathlib.html#module-pathlib